src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import java.io.*;
```

Comment:
CheckStyle convention recommends to import one by one

Reply:
Fixed.